### PR TITLE
Add tips & tricks section with initial entries

### PR DESCRIPTION
**Pull request by atifaziz**
This PR addresses #4.

I've made two initial entries. Something that is up for discussion is that whether the tweets should be embedded or just linked. Not embedding doesn't make sense as tweets don't have a title and coming up with a paraphrased summary might not be shorter than the tweet itself. Also, if tweets are embedded then one can just find them by some text search in the browser (e.g. using usually <kbd>CTRL</kbd>+<kbd>F</kbd> in a browser on Windows). However, one downside I see over time is that the section can grow large and overpower others so perhaps it should be added at the bottom then?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,31 @@ Template for github repo links:
 - [ValueObjectGenerator](https://github.com/RyotaMurohoshi/ValueObjectGenerator)  - ![stars](https://img.shields.io/github/stars/RyotaMurohoshi/ValueObjectGenerator?style=flat-square&cacheSeconds=604800) ![last commit](https://img.shields.io/github/last-commit/RyotaMurohoshi/ValueObjectGenerator?style=flat-square&cacheSeconds=86400) C# source generator is for ValueObjects (ie.Wrapper classes).
 - [WrapperValueObject](https://github.com/martinothamar/WrapperValueObject)  - ![stars](https://img.shields.io/github/stars/martinothamar/WrapperValueObject?style=flat-square&cacheSeconds=604800) ![last commit](https://img.shields.io/github/last-commit/martinothamar/WrapperValueObject?style=flat-square&cacheSeconds=86400) - for creating simple value objects wrapping primitive types.
 
+## Tips & Tricks
+
+Collection of tips and tricks (simple and brief to fit in Tweet):
+
+> See files emitted by [#SourceGenerators] by adding these properties to your (*.csproj) project file:
+>
+> ```c#
```
Comment (amis92) [inline README.md:47]: it's xml here, actually :)

**Comment by amis92** [comment]
We'll think when it grows ;) Right now, that's perfect. Thanks!

**Comment by ironcev** [comment]
Embedded at the bottom would also be my preference.

File: README.md (hunk 1 above)
Comment (atifaziz) [reply README.md]: 🤦🏼

**Comment by atifaziz** [comment]
I have made a slight change in layout with bb3e10587426fb2c31e6fefcb88af88b4087250d, where the source info appears before the quoted tweet. It provides better visual space & separation between tweets. I can revert this if there's a strong preference for the original version.

**Comment by amis92** [comment]
Thanks!

**Pull request merged**